### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by nopara73** [molnard/WasabiCoinGraph#1]
```
/mnt/c/Users/user/Desktop/WasabiCoinGraph$ ./wcli.sh selectwallet Wallet
./wcli.sh: line 2: $'\r': command not found
./wcli.sh: line 3: syntax error near unexpected token `$'{\r''
'/wcli.sh: line 3: `function config_extract() {
```

**Comment by molnard** [comment]
https://github.com/molnard/WasabiCoinGraph/commit/f1fc9ee324437801054cefee8ee9713e45189b25

**Comment by molnard** [comment]
Solved?

**Comment by nopara73** [comment]
Nope
```
./wcli.sh: line 2: $'\r': command not found
./wcli.sh: line 3: syntax error near unexpected token `$'{\r''
'/wcli.sh: line 3: `function config_extract() {
```

**Comment by molnard** [comment]
Can you clone the repo again?

**Comment by molnard** [comment]
Sorry, my bad try it now. https://github.com/molnard/WasabiCoinGraph/commit/a7870f824524ce354cfae2a057116a47977bd125

**Comment by nopara73** [comment]
Still not :*(

```sh
./wcli.sh: line 2: $'\r': command not found
./wcli.sh: line 3: syntax error near unexpected token `$'{\r''
'/wcli.sh: line 3: `function config_extract() {
```

**Comment by molnard** [comment]
The end of the line should be LF. If it is CRLF it will fail like above. I did a fresh clone and it is working for me. 

https://www.atechtown.com/notepad-eol-conversion/

TLDR

Go to Edit-> EOL Conversion – > Unix(LF)

**Comment by nopara73** [comment]
Did a fresh clone and this works.
